src/coercions/string.rs

```diff
@@ -25,3 +26,11 @@ impl ToRust<String> for CheckedValue<String> {
         unsafe { std::str::from_utf8_unchecked(slice) }.to_string()
     }
 }
+
+impl ToRuby for String {
+    fn to_ruby(self) -> VALUE {
+        let len = self.len();
+        let cstr = CString::new(self).unwrap();
```

Comment:
`rb_utf8_str_new` calls `str_new0` which [copies the string](https://github.com/ruby/ruby/blob/trunk/string.c#L706) we provide, so we retain ownership and shouldn't be forgetting it (otherwise we'll leak).